necroHeal.lic

```diff
@@ -0,0 +1,57 @@
+=begin
+
+  Documentation: https://elanthipedia.play.net/Lich_script_development#necroheal
+  ## ** Placeming comments here until finalized and added to pedia.
+
+  Necromancer stand-alone healing script
+  
+  Uses all settings defined in HEAL YAML FILE (default)
+  Consume logic via combat trainer
+  
+  TO DO:
+  - See pull request #1462 for harvested materials.
+  - See if there is a background way to use common-healing to determine wound status
+  - Add YAML variable to swap YAML file name
+  
+=end
+
+custom_require.call(%w(common common-healing events))
+
+
+  if DRCH.check_health.empty?

```

Comment:
Should this use the `saferoom_health_threshold` setting rather than healing on *any* wounds? What about poison or bleeding?

See https://github.com/rpherbig/dr-scripts/blob/master/safe-room.lic#L47 for an example of what I men.